test/specs/functional/keyboardNav.js

```diff
@@ -329,6 +329,33 @@ describe("Keyboard navigation functional browser driven tests", function() {
 			});
 		});
 
+		describe("Given I am on a nice service", function () {
+			it("I can use the skip to main content link when main content ID is not provided", async function() {
+				browser.url("/example.niceorg.html");
+				browser.keys("Tab");
```

Comment:
Relying on a single tab key to get to an element seems be flaky cross-browser, which is why we intriduced the focusOnLogo function - probably better to use this and then shift tab back

Reply:
Hey this isn't actually focusing on the logo as skip-to-main-content is first in the tabbing order so I am in fact tabbing to this.